compilers/concrete-optimizer/v0-parameters/src/bin/v0-parameters-by-level.rs

```diff
@@ -28,16 +28,18 @@ fn main() {
     }
 
     for security_level in supported_security_levels() {
-        let filename_date = if args.wop_pbs {
-            format!("ref/wop_pbs_{year}-{month}-{day}_{security_level}")
-        } else {
-            format!("ref/v0_{year}-{month}-{day}_{security_level}")
-        };
-        let filename_last = if args.wop_pbs {
-            format!("ref/wop_pbs_last_{security_level}")
+        let ap_type = if args.wop_pbs { "wop_pbs" } else { "v0" };
+
+        let suffix = if args.ciphertext_modulus_log == 64 && args.fft_precision == 53 {
+            "".to_owned()
         } else {
-            format!("ref/v0_last_{security_level}")
```

Comment:
On peut garder le last non comme sur les autres cas ?

Reply:
I don't understand what you mean